### PR TITLE
Added Danish 🇩🇰

### PR DESCRIPTION
**Pull request by dinsen**

### CODE REVIEW

File: app/da.strings [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+/* ERRORS */
+"Something went wrong" = "Noget gik galt";
+
+"Please try again later" = "Prøv venligst igen senere"; 
+"Not enough translation tokens" = "Ikke nok oversættelses tokens";
+"Get more" = "Få mere";
+"Translation Error" = "Oversættelsesfejl";
+"No purchases could be found for your account" = "Der blev ikke fundet nogen køb på din konto";
+
+"Almost ready!" = "Næste klar!";
```
Comment (dinsen) [inline app/da.strings]: I saw this when onboarding, but it's not in the en.strings file.

**Comment by jordibruin** [comment]
Thanks about adding the errors, I've added a few more strings and will update the strings file later today! Thanks for trying stuff out 🙂

**Comment by jordibruin** [comment]
I've just updated the English strings file, could you add your localization in there?

**Comment by dinsen** [comment]
@jordibruin it's updated now 🚀

**Comment by jordibruin** [comment]
You're the best @Dinsen

**Pull request merged**